commands/show_cluster.go

```diff
@@ -250,6 +251,11 @@ func showClusterRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
 
 	output = append(output, color.YellowString("Kubernetes API endpoint:")+"|"+clusterDetails.APIEndpoint)
 
+	if len(clusterDetails.AvailabilityZones) > 0 {
+		sort.Strings(clusterDetails.AvailabilityZones)
+		output = append(output, color.YellowString("Availability Zones:")+"|"+strings.Join(clusterDetails.AvailabilityZones, ","))
```

Comment:
```suggestion
		output = append(output, color.YellowString("Availability Zones:")+"|"+strings.Join(clusterDetails.AvailabilityZones, ", "))
```

Aligning this with instance types/vm sizes in the `gsctl info` command